plugins/operators/gsheets_rollout.py

```diff
@@ -38,13 +39,18 @@ def convert_rollout_week_sheet_name(date_with_week_number: str) -> datetime.date
 
 def convert_sheet_data_into_feature_subnet_map(
     sheet_data: list[list[Any]],
+    log: Logger
 ) -> dict[SubnetId, FeatureName]:
     headings, rows = sheet_data[0], sheet_data[1:]
     if not headings:
         raise Warning("No headings on sheet, ignoring")
     feature_names = headings[1:]
     subnet_id_feature_map: dict[SubnetId, FeatureName] = {}
-    for row in rows:
+    for (index, row) in enumerate(rows):
+        log.info("Checking row %s" % row)
+        if not len(row):
```

Comment:
How about if we stop iterating if cell (col) 0 is empty?

Reply:
That means that an empty line in the middle would cut out the rollout. I'd rather raise and error or stop. The issue is that if you check Week 45 you will see there are no empty rows.

Reply:
An empty line in the middle would not be correct my point of view. But I see your point.
Q: is there a way to make a comment (as a text) somewhere in the sheet?
Can we skip a row if cell 0 (col 0) is empty, and we then just proceed to the next row?